### DISCUSSION TITLE
Incorrect show dates and/or times

### DISCUSSION THREAD

**Discussion by johnmc98683** [pymedusa/Medusa#10318] [Q&A]
Some, but not all, shows in my list are listed as being aired on the wrong day and/or wrong time.
For instance, NCIS is listed as being aired on Sunday, at 9 pm. It is actually aired on Monday. NCIS Hawaii is listed as airing at 7 pm, instead of 8 pm.
The configured timezone is PST, which is correct.
These errors are shown for both the last airing and the next airing.

John

**Comment by p0psicles** [comment]
Which show indexer do you use for them?

**Comment by johnmc98683** [comment]
All shows are using imdb/thetvdb except for NCIS, which uses imdb/themoviedb,
oddly enough.

**Reply by p0psicles** [reply]
Themoviedb doesn't have airtime. So it's fixed to 00:00. Tvdb and tvmaze should be accurate

**Comment by johnmc98683** [comment]
The ability to edit indexers would help solve the problem, perhaps. And would be
less of a headache than deleting the show and installing it again. I won't be
selecting "All Indexers" again, while adding new shows.
And I understand the ability to edit indexers in the development stage.
